apps/redis-secrect-provisioning-example/redis-server.manifest

```diff
@@ -51,6 +51,7 @@ sgx.trusted_files.libnssdns = file:../../deps/graphene/Runtime/libnss_dns.so.2
 sgx.trusted_files.libresolv = file:../../deps/graphene/Runtime/libresolv.so.2
 sgx.trusted_files.librt = file:../../deps/graphene/Runtime/librt.so.1
 sgx.trusted_files.libgccs = file:/lib/x86_64-linux-gnu/libgcc_s.so.1
+sgx.trusted_files.libmdns4 = file:/lib/x86_64-linux-gnu/libnss_mdns4_minimal.so.2
```

Comment:
This lib exists on Ubuntu 16.04 but not on Ubuntu 18.04. I believe this Redis example was tested on Ubuntu 18.04. So there are two options: (1) create a separate manifest called `redis-server-16.04.manifest` and choose between the two in the Makefile, or (2) add this line as a comment here, explaining that it is needed for Ubuntu 16.04.

Reply:
It would report the

> "Accessing file:/lib/x86_64-linux-gnu/libnss_mdns4_minimal.so.2 is denied. This file is not trusted or allowed"

on Ubuntu 18.04, not 16.04 so that's why I added it directly into original manifest file since it should still work without this lib on Ubuntu 16.04

Reply:
I am confused by this explanation. Under Ubuntu 18.04, this added line produces a Python error:
```
Exception: Cannot resolve file:/lib/x86_64-linux-gnu/libnss_mdns4_minimal.so.2 or the file does not exist.
```
and the build fails.

Reply:
When trying to run this, I actually only receive an error about a missing libnss-myhostname.so; on Ubuntu 16.04 and 18.04. Seems like this particular library must now be installed explicitly, whereas previously it was pulled in by some other Ubuntu package. That's my theory at least. Regarding libnss_mdns4_minimal.so, it seems like it is not actually required, so I am not sure why it is needed in your setup. I can run Redis-server just fine without adding this line to the manifest.